js/index.d.ts

```diff
@@ -0,0 +1,87 @@
+/*
+  Typings for @sentry/cli
+*/
+declare module '@sentry/cli' {
+  export interface SentryCliOptions {
+    url?: string;
+    authToken?: string;
+    apiKey?: string;
+    dsn?: string;
+    org?: string;
+    project?: string;
+    vscRemote?: string;
+    silent: boolean;
+  }
+
+  export interface SentryCliUploadSourceMapsOptions {
+    include: string | string[];
+    ignore?: string[];
+    ignoreFile?: string | null;
+    rewrite?: boolean;
+    sourceMapReference?: boolean;
+    stripPrefix?: string[];
+    stripCommonPrefix?: boolean;
+    validate?: boolean;
+    urlPrefix?: string;
+    urlSuffix?: string;
+    ext?: string[];
+  }
+
+  export interface SentryCliNewDeployOptions {
+    env: string;
+    started?: number;
+    finished?: number;
+    time?: number;
+    name?: string;
+    url?: string;
+  }
+
+  export interface SentryCliCommitsOptions {
+    auto?: boolean;
+    repo?: string;
+    commit?: string;
+    previousCommit?: string;
+  }
+
+  export interface SentryCliReleases {
+    ['new'](
```

Comment:
is `new` reserved keyword here and we cannot use `new(release: string...`? 😶

Reply:
yes~